content/blog/2023/when-to-choose-a-responsive-web-app.md

```diff
@@ -223,8 +223,10 @@ mobile device's GPS for geofencing or access to light and proximity sensors,
 then an RWA won't work.
 
 It is *possible* to access some advanced device features in the context of an
-RWA, but then you would be moving into the [progressive web
-app](/2023/07/05/when-to-choose-a-progressive-web-app/) space.
+RWA, but then you would be moving into the [progressive web app][pwa] space.
+
+<!-- [pwa]: /2023/09/01/when-to-choose-a-progressive-web-app/ -->
+[pwa]: https://web.dev/progressive-web-apps/
```

Comment:
All the other changes here are hidden for now, but this fixes one visible broken link.